index.html

```diff
@@ -16,7 +16,7 @@
 
     <section class="section3">
           
-          <img src="images/c.png" alt="ada" class="imagesec4"></div>
+          <img src="images/c.png" alt="ada" class="imagesec4">
          
         	  <h2 class="tonic tonic1">Tonic</h2>

```

Comment:
- [ ] The navbar(toolbar) and the headline section are not in your HTML. Kindly include it. 

- [ ] I noticed that you have not implemented all that the other reviewers have requested. Please endeavor to implement/fix the issues and re-submit for review. 

You can see below that your design is far from the Figma design. See the links to previous helpful comments by other reviewers:
- [Click here to see the review.](https://github.com/azizunnabi/PORT-MOB-PROJ/pull/1#pullrequestreview-1033549224). Please begin from there a fix accordingly through every comment to the end. 

Your design | Figma
--- | ---
| ![image](https://user-images.githubusercontent.com/84156714/185718859-801ef29b-4441-43b2-bab3-aae524c44318.png) | ![](https://github.com/microverseinc/curriculum-html-css/raw/main/portfolio/images/m1_setup/about-myself-mobile.png)